### PR TITLE
Sessions testing

### PR DESCRIPTION
**Pull request by mullermp**
Adds testing for dynamodb session store

### CODE REVIEW

File: test/dummy/config/dynamo_db_session_store.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Uncomment and manipulate the key value pairs below
```
Comment (alextwoods) [inline test/dummy/config/dynamo_db_session_store.yml:1]: Should this not be checked in?  Assume its getting generated by the tests

File: test/dummy/db/migrate/20201112193021_create_dynamo_db_sessions_table.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class CreateDynamoDbSessionsTable < ActiveRecord::Migration[6.0]
```
Comment (alextwoods) [inline test/dummy/db/migrate/20201112193021_create_dynamo_db_sessions_table.rb:1]: Ditto on this - I think we'd end up with a new file each time from the test w/ differnt date stamps?

**Review by alextwoods — APPROVED** [review]

File: test/dummy/db/migrate/20201112193021_create_dynamo_db_sessions_table.rb (hunk 2 above)
Comment (mullermp) [reply test/dummy/db/migrate/20201112193021_create_dynamo_db_sessions_table.rb:1]: Rails is smart enough to check the contents (and sometimes the name) so even running at a later date will determine that the file is identical and not create a new one.

File: test/dummy/config/dynamo_db_session_store.yml (hunk 1 above)
Comment (mullermp) [reply test/dummy/config/dynamo_db_session_store.yml:1]: I dumped these files in /dummy/ so they could be used with the config loading tests (action_dispatch/session)

**Review by alextwoods — APPROVED** [review]

**Pull request merged**